Sources/MapboxNavigation/NavigationComponent.swift

```diff
@@ -37,7 +37,9 @@ public protocol CarPlayConnectionObserver: class {
  */
 public protocol NavigationStatusPresenter: class {
     /**
-     Shows the status view for a specified amount of time.
+     Shows a Status for a specified amount of time.
      */
-    func showStatus(title: String, spinner: Bool, duration: TimeInterval, animated: Bool, interactive: Bool)
+    func addNewStatus(status: StatusView.Status)
```

Comment:
This is technically a backwards-incompatible change, because an application may expect to be able to call `showStatus(title:spinner:duration:animated:interactive:)`. In order for these changes to go into v1._x_, the old method needs to be left in place but marked as deprecated. Its implementation inside StatusView can create a Status and pass it into the new `show(_:)` method.